Deploy-CAM.ps1

```diff
@@ -2111,24 +2142,25 @@ else {
             # entered a name. Let's see if it matches any resource groups first
             $rgMatch = $resouceGroups | Where-Object {$_.ResourceGroupName -eq $rgIdentifier}
             if ($rgMatch) {
-                $rgName = $rgMatch.ResourceGroupName
-                Write-Host ("Resource group `"$rgName`" already exists. The current one will be used.")
+                Write-Host ("Resource group `"$($rgMatch.ResourceGroupName)`" already exists. The current one will be used.")
                 $selectedRGName = $true
             }
             else {
                 # make a new resource group and on failure go back to RG selection.
-                $rgName = $rgIdentifier
```

Comment:
The other way to solve the rgName issue is to get rid of the variable so we don't need to keep remembering to set it :) @cptera - I think this affects your PR as well.